src/env.h

```diff
@@ -52,6 +52,7 @@
 #include <list>
 #include <unordered_map>
 #include <unordered_set>
+#include <set>
```

Comment:
Thanks for the IWYU improvement! I think we are missing some more STL includes for this file and you can run `git grep -oh "std::\w*" src/env.h | sort | uniq` to get a list of all the `std::` symbols we use here. Could you please add the other includes too?

Reply:
Ok, I did IWYU for STL files on `src/env.h` and updated the commit message/title to be more accurate since it's more than just removing some unused includes now.

Reply:
Could you please do it for src/util.h too as we are touching that file in this PR?

Reply:
Done